docs/content/new-tutorial/your-first-asset.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: "Intro to the Dagster Tutorial: Your First Asset | Dagster Docs"
+description: "Get set up for the intro to assets tutorial."
+noindex: True
+---
+
+# Writing Your Pipeline with Assets
+
+Now that you've installed Dagster and know what assets are, it's time for you to build parts of your first data pipeline!
+
+In this tutorial, you will analyze activity on a popular news aggregation website called Hacker News. Similar to common data pipelines, you will fetch data from the website, clean it up, and build a report summarizing some findings.
+
+By the end of this section, you will:
+
+- ingest data from the Hacker News API
+- turn that ingestion into an asset
+- explore the Dagit, the Dagster UI
+- launch your first Dagster run manually
+
+## Understanding the pipeline
+
+Hypothetically, you have a Python script called `analyze_hackernews.py`. The code for it is below. If you'd like to see this code work, you can run:
```

Comment:
A few thoughts on this Python script approach:
- I think it needs a little more explanation / framing.  We should briefly motivate why we're showing them a Python script instead of jumping straight into Dagster.
- The script does not actually generate any assets - i.e. persistent objects in storage.  I think we need to add some `pickle.dumps` calls or similar to make it analogous to what we end up building in Dagster.
- The script is long and it's a lot to digest. What do you think about first just showing the part that generates the results DataFrame and adding the other part in a later section?